Стилизуем письма

Nitraatss opened the pull request:
---
:mortar_board: [Стилизуем письма](https://up.htmlacademy.ru/email/1/user/334301/tasks/4)

Nitraatss committed:
Добавлена базовая стилизация для первого письма

Nitraatss committed:
Добавлена базовая стилизация для второго письма. Исправление разметки в первом письме.

Nitraatss committed:
Update index.html

Nitraatss committed:
Добавлены базовые стили 3 письма

Nitraatss committed:
FIx

ricomen commented on 1-invitation/index.html:
```diff
@@ -8,256 +8,424 @@
-  <body>
-    <div>
+  <body style="width: 100%; padding-left: 0px; padding-right: 0px; padding-top: 0px; padding-bottom: 0px; Margin: 0;">
+    <div style="Margin: 0; color: #fff; display: none !important; font-family: arial; font-size: 1px; line-height: 0;">
```
color: #fff; не используем короткую запись

ricomen commented on 1-invitation/index.html:
```diff
@@ -8,256 +8,424 @@
+                    <tr>
+                      <td align="center" style="Margin: 0; padding-top: 32px; padding-bottom: 16px; padding-left: 16px; padding-right: 16px;">
+                        <picture>
+                          <img src="./img/logo.png" srcset="./img/logo@2x.png 2x" width="140" height="24" alt="PlayHTML" border="0" />
```
srcset не поддерживается используем x2 картинку сразу в src

ricomen commented on 1-invitation/index.html:
```diff
@@ -8,256 +8,424 @@
+                          </tr>
+                          <tr>
+                            <td align="left" style="Margin: 0; padding-top: 0px; padding-bottom: 0px; padding-left: 0px; padding-right: 0px;">
+                              <p><a href="#" target="_blank" title="Переход на страницу бесплатных вебинаров">Бесплатные вебинары</a></p>
```
3.8. Полезные ссылки в подвале (бесплатные вебинары, обучающие курсы, тесты, наш блог, истории студентов) являются ссылками на https://htmlacademy.ru/.

ricomen commented on 2-welcome/index.html:
```diff
@@ -8,260 +8,368 @@
-  <body>
-    <div>
+  <body style="idth: 100%; padding-left: 0px; padding-right: 0px; padding-top: 0px; padding-bottom: 0px; Margin: 0;">
+    <div style="Margin: 0; color: #fff; display: none !important; font-family: arial; font-size: 1px; line-height: 0;">
```
color: #fff color: #ffffff

ricomen commented on 2-welcome/index.html:
```diff
@@ -8,260 +8,368 @@
+                          </tr>
+                          <tr>
+                            <td align="left" style="Margin: 0; padding-top: 0px; padding-bottom: 0px; padding-left: 0px; padding-right: 0px;">
+                              <p><a href="#" target="_blank" title="Переход на страницу бесплатных вебинаров">Бесплатные вебинары</a></p>
```
3.8. Полезные ссылки в подвале (бесплатные вебинары, обучающие курсы, тесты, наш блог, истории студентов) являются ссылками на https://htmlacademy.ru/.

ricomen commented on 3-promo/index.html:
```diff
@@ -8,462 +8,712 @@
-    <div>
-      <table width="100%" cellspacing="0" cellpadding="0" border="0">
+  <body style="width:100%; padding:0; Margin:0;">
+    <div style="margin:0; color:#fff; display:none !important; font-family:arial; font-size:1px; line-height:0;">
```
color:#ffffff;

Nitraatss committed:
Правки

Удален тэг picture.
Изображения 2x.
Обновлены полезные ссылки.
Исправление цветов.
